packages/deprecated-component-library-helpers/styles/type.scss

```diff
@@ -0,0 +1,359 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/typography";
+
+$ca-grid: 1.5rem; // 24px @ default root font-size of 16px
+
+$ca-weight-extra-light: 200;
+$ca-weight-light: 300;
+$ca-weight-book: 400;
+$ca-weight-medium: 500;
+$ca-weight-semibold: 500; // Note: in Sketch, semibold is 600. But murmur has an existing value of semibold=500 that is heavily used.
+
+// Combine these into a Sass map ($ca-default-font) once node-sass includes libsass 3.5.0.beta.3 with this bug fix: https://github.com/sass/libsass/issues/2309
+$ca-default-font-family: "Open Sans", Helvetica, Arial, sans-serif;
+$ca-default-font-base-size: 0.875rem; /* 14px */
+$ca-default-font-descender-height: 0.115;
+
+// Combine these into a Sass map ($ca-default-font) once node-sass includes libsass 3.5.0.beta.3 with this bug fix: https://github.com/sass/libsass/issues/2309
+$ca-ideal-sans-font-family: "Ideal Sans A", "Ideal Sans B",
+  $ca-default-font-family;
+$ca-ideal-sans-font-base-size: 1rem; /* 16px */
+$ca-ideal-sans-font-descender-height: 0.14;
+
+$ca-greycliff-font-base-size: 1rem; /* 16px */
+$ca-greycliff-font-descender-height: 0.098;
```

Comment:
This is the value I created by hand. I'm sure there's a way to get this mathematically perfect, but this seems close enough.

Reply:
![image](https://user-images.githubusercontent.com/6406263/74629252-e4e9b380-51ab-11ea-8549-baf8e5fd3797.png)
I don't think I understand how the formula works 🤔 I thought it would be the `descender height in units`/`units per em`, which for Greycliff would be `389/2048 = 0.1899`. That looks quite different to what you got by hand, which makes me think that these metrics I mentioned here don't represent what I thought they represented. Hmmm